cypress/e2e/swap/errors.test.ts

```diff
@@ -13,12 +13,16 @@ describe('Swap errors', () => {
       // Stub the wallet to reject any transaction.
       cy.stub(hardhat.wallet, 'sendTransaction').log(false).rejects(new Error('user cancelled'))
 
-      // Attempt to swap.
-      cy.get('#swap-currency-output .token-amount-input').clear().type('1').should('have.value', '1')
+      // Enter amount to swap
+      cy.get('#swap-currency-output .token-amount-input').type('1').should('have.value', '1')
       cy.get('#swap-currency-input .token-amount-input').should('not.have.value', '')
+
+      // Submit transaction
       cy.get('#swap-button').click()
       cy.get('#confirm-swap-or-send').click()
+      cy.wait('@eth_estimateGas')
```

Comment:
tell me about this

Reply:
Calls to the wallet are now aliased, using `@${METHOD_CALLED}`, so this waits for the wallet to call `eth_estimateGas`.
Adding this throughout the `swap/` tests makes the tests more comprehensive, but also gives hardhat more time to resolve these requests, because each time we call `cy.wait` the test is given a 5s request timeout and 30s response timeout.

Reply:
The click should resolve immediately, and `wait` detects the request sent out. It's the response that hangs.
This should be fixed by caching hardhat archive calls, done in #6650.